lint/highlight.py

```diff
@@ -400,6 +433,28 @@ def draw(self, view):
                 region = sublime.Region(self.newlines[line], self.newlines[line])
                 gutter_regions[error_type].append(region)
 
+        def hide_phantoms(href):
```

Comment:
this will not work because `hide_phantoms` require the `href` attr(i don't know why)